Bottom: 00ad0524f7341b2347d1f9562ce879d830490c09
Top:    976a2828c389f5dc01ab86fca3d303547c33e8a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-21 00:03:35 +0000

Refresh of add-vrml-model-rendering-suppo

---

diff --git a/src/hid/common/hidgl_material.c b/src/hid/common/hidgl_material.c
index 766ae8d..7709a44 100644
--- a/src/hid/common/hidgl_material.c
+++ b/src/hid/common/hidgl_material.c
@@ -66,7 +66,6 @@ struct _hidgl_material {
 };
 
 
-/* If either vs or fs is NULL, used the fixed function pipeline for that */
 hidgl_material *
 hidgl_material_new (char *name)
 {
